src/state/timer-state.js

```diff
@@ -166,6 +172,28 @@ class TimerState {
     this.publishConfig()
   }
 
+  setClearClipboardHistoryBetweenTurns(value) {
+    this.clearClipboardHistoryBetweenTurns = value
+    this.publishConfig()
+  }
+
+  clearClipboardHistory() {

```

Comment:
I think this whole method, plus the `let clipboardy = require('clipboardy')` from the top (shouldn't it be `const`?) should be moved into a separate file. It's really not cohesive with the rest of the file. Maybe move to a `clear-clipboard.js` instead?

Reply:
Thank you - it should be `const` and a separate file makes sense, will change that.